pkg/steps/output_image_tag_test.go

```diff
@@ -38,8 +38,7 @@ func TestOutputImageStep(t *testing.T) {
 			api.InternalImageLink("configToAs"),
 		},
 		provides: providesExpectation{
-			params: map[string]string{"IMAGE_CONFIGTOAS": "uri://somewhere:configToTag"},
-			link:   api.ExternalImageLink(config.To),
+			params: map[string]string{"IMAGE_CONFIGTOAS": "uri://somewhere@fromImageName"},
```

Comment:
I don't think the mock actually updates the spec for the ImageStream when we create the ImageStreamTag, and we can always get away with reference by digest instead of floating tag, so this is both a semi-broken test but also a change in behavior to share the func we use to resolve digests